instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -479,13 +479,13 @@ func peerAttr(addr string) []attribute.KeyValue {
 	var attr []attribute.KeyValue
 	if ip := net.ParseIP(host); ip != nil {
 		attr = []attribute.KeyValue{
-			semconv.NetSockPeerAddr(host),
-			semconv.NetSockPeerPort(port),
+			semconv.NetworkPeerAddress(host),
+			semconv.NetworkPeerPort(port),
 		}
 	} else {
 		attr = []attribute.KeyValue{
-			semconv.NetPeerName(host),
-			semconv.NetPeerPort(port),
+			semconv.NetworkPeerAddress(host),
+			semconv.NetworkPeerPort(port),
```

Comment:
These are breaking changes. We should either mark it explicitly in the changelog, or do as we do in otelhttp and provide a smooth migration path.
See https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5132